release-notes/opensearch.job-scheduler.release-notes-1.2.0.0.md

```diff
@@ -0,0 +1,23 @@
+## Version 1.2.0.0 2021-11-05
+
+Compatible with OpenSearch 1.2.0
+
+### Features
+
+* Adds a delay parameter to the job scheduler ([#61](https://github.com/opensearch-project/job-scheduler/pull/61))
+
+### Bug Fixes
+
+* Fixes possible negative jitter ([#78](https://github.com/opensearch-project/job-scheduler/pull/78))
+
+### Infrastructure
+
+* Publish MD5 and SHA1 signatures ([#71](https://github.com/opensearch-project/job-scheduler/pull/71))
```

Comment:
This was part of 1.1 already

Reply:
I believe the same applies here as my comment for the delay parameter.